docs/list.md

```diff
@@ -312,6 +312,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: No special signup known.
 - **Notes**: Read more at [JavaScript Today](https://blog.javascripttoday.com/blog/hacktoberfest-2023/).
 
+### K
+
+#### **KodaDot**
+
+- **Swag**: Cash Prizes not disclosed yet.
```

Comment:
```suggestion
- **Swag**: Cash Prizes
```
Bounties for those issues are tagged accordingly. So swag as cash prizes, should be okay.

Reply:
updated it